src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/DependencyTreeTelemetryService.cs

```diff
@@ -178,7 +178,7 @@ public void Dispose()
                 // Scan the snapshot and tally dependencies
                 foreach ((TargetFramework targetFramework, TargetedDependenciesSnapshot targetedSnapshot) in _dependenciesSnapshot.DependenciesByTargetFramework)
                 {
-                    var targetData = new TargetData(targetFramework.ShortName);
+                    var targetData = new TargetData(targetFramework.TargetFrameworkMoniker);
```

Comment:
@jjmew this will potentially change the representation of TFM in telemetry data.

i.e. before `net472`, after `.NETFramework,Version=v4.7.2`.